lib/pr_checker.js

```diff
@@ -374,9 +373,10 @@ export default class PRChecker {
 
     // GitHub new Check API
     for (const { status, conclusion, app } of checkSuites.nodes) {
-      if (app && IGNORED_CHECK_SLUGS.includes(app.slug)) {
+      if (app.slug !== 'github-actions') {
```

Comment:
```suggestion
      if (app && app.slug !== 'github-actions') {
```

Reply:
```suggestion
      if (app?.slug !== 'github-actions') {
```

a suggestion upon your suggestion

Reply:
I'm not sure this is needed.